ppafm/fieldFFT.py

```diff
@@ -287,7 +287,7 @@ def potential2forces_mem( V, lvec, nDim, sigma = 0.7, rho=None, multipole=None,
         E         = np.real(np.fft.ifftn(convFFT * (dd[0]*dd[1]*dd[2]) / (detLmatInv) ) )
     if doForce:
         if(verbose>0): print('--- Get Forces ---')
-        convFFT  *= 2*np.pi*1j*(dd[0]*dd[1]*dd[2]) / (detLmatInv)
+        convFFT  *= -2*np.pi*1j*(dd[0]*dd[1]*dd[2]) / (detLmatInv)
```

Comment:
Above in the `generateElFF.py` the potential is first flipped (` V *= -1`) because of wrong units like I mentioned in the comment on the issue https://github.com/Probe-Particle/ppafm/issues/112#issuecomment-1479034659, so in total the electrostatics stay the same.